src/Tribe/Events/Attendees_List.php

```diff
@@ -68,20 +90,31 @@ public function is_event_hiding_attendee_list( $post ) {
 	 * @since TBD
 	 *
 	 * @param bool $should_hide Whether the optout form should be hidden or not.
+	 * @param int  $post_id     The post ID the ticket belongs to.
 	 *
 	 * @return bool Whether we should hide the optout option.
 	 *
 	 * @see \Tribe\Tickets\Events\Events_Service_Provider::hooks
 	 */
-	public function should_hide_optout( $should_hide ) {
-		try {
-			global $post;
+	public function should_hide_optout( $should_hide, $post_id = 0 ) {
+		global $post;
+
+		return true;
+
+		if ( empty( $post_id ) ) {
```

Comment:
:warning: Code after the RETURN statement on line 102 cannot be executed

<sub>hash: 827c2e16ce36a1df2ededb00752c6a40</sub>